src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,141 @@
+
+**Key Vault**
+
+* Refine error messages for HSM `list-deleted` and `purge` (#15657)
+* Support selective key restore for managed HSMs (#15867)
+
+**NetAppFiles**
+
+* [BREAKING CHANGE] az netappfiles pool update: Removed service-level from parameters. (#15602)
+* `az netappfiles pool update`: Added optional parameter qos-type. (#15602)
+* `az netappfiles pool create`: Added optional parameter qos-type. (#15602)
+* `az netappfiles volume replication suspend`: Added force-break-replication as optional parameter. (#15602)
+* Add az netappfiles volume replication re-initialize: New command added to re-initialise replication. (#15602)
+* Add az netappfiles volume pool-change: New command to change the pool of a volume. (#15602)
+* Add az netappfiles snapshot policy: New command group with list, delete, update, show, create and volumes commands. (#15602)
+* Add az netappfiles account backup: New command group with show, list and delete commands (#15602)
+* Add az netappfiles volume backups: New command group with show, list, delete, update and create commands. (#15602)
+* Add az netappfiles account backup-policy: New command group with show, list, delete, update and delete commands. (#15602)
+* Add az netappfiles vault list: New command added. (#15602)
+* `az netappfiles account ad add`: Added optional parameters kdc-ip, ad-name, server-root-ca-certificate and backup-operators (#15602)
+* `az netappfiles volumes create`: Added optional parameters snapshot-policy-id, backup-policy-id, backup-enabled, backup-id, policy-enforced, vault-id, kerberos-enabled, throughput-mibps, snapshot-directory-visible, security-style, kerberos5-read-only, kerberos5-read-write, kerberos5i-read-only, kerberos5i-read-write, kerberos5p-read-only, kerberos5p-read-write and has-root-access. (#15602)
+* `az netappfiles volume update`: Added optional parameters vault-id, backup-enabled, backup-policy-id, policy-enforced and throughput-mibps (#15602)
+
+**Network**
+
+* Fix bug that can't create a Standard_v2 application-gateway without a private static IP address (#15757)
+* `az network dns zone import`: Raise FileOperationError instead of FileNotFoundError if zone file doesn't exist (#15870)
+* Fix NoneType error crash while deleting nonexisting resources of ApplicationGateway, LoadBalancer, Nic (#15886)
+
+**Private DNS**
+
+* `az network private-dns zone import`: Raise FileOperationError instead of FileNotFoundError if zone file doesn't exist (#15874)
+
+**Profile**
+
+* `az login`: Add back the warning that a browser is opened (#14505)
+
+**Role**
+
+* `az role assignment create`: Make description, condition, condition_version preview (#15690)
+
+**Security**
+
+* `az security pricing`: Update help to reflect current API version being called (#15807)
+
+**Storage**
+
+* Fix issue #15600: az storage fs exists: in case fs does not exist ResourceNotFoundError is returned (#15643)
+* Fix issue #15706: The examples for storage container create are incorrect (#15731)
```

Comment:
```suggestion
* Fix #15600: az storage fs exists: in case fs does not exist ResourceNotFoundError is returned (#15643)
* Fix #15706: The examples for storage container create are incorrect (#15731)
```